charts/testkube-api/values.yaml

```diff
@@ -53,6 +53,12 @@ image:
 nameOverride: ""
 fullnameOverride: ""
 
+## Custom job-container-template.yml that will passed to Testkube API
+jobContainerTemplate: ""
```

Comment:
@bkomraz1 thank you for the PR! Could you please duplicate values with the explanations above to the `testkube/values.yaml` file?

Reply:
hey @bkomraz1!
Is this PR still relevant?

Reply:
yes

Reply:
@ypoplavs Please review